src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+  currentContent = tabs[0].content,
```

Comment:
```suggestion

```

Reply:
Consider using the find array method to find tab content and check if the tab exists

Reply:
Done but I'm not sure...